### PR TITLE
Change metric name of http request/response

### PR DESCRIPTION
**Pull request by kojilin**
Apply this change will make metric name from GET/POST to ${path}#GET/${path}#POST.

### CODE REVIEW

**Comment by trustin** [comment]
How about using `/` or `#` as a separator between method name and path?

**Comment by kojilin** [comment]
Thanks, I think `#` is good for separator.

**Comment by trustin** [comment]
@blmarket PTAL as well.

File: src/test/java/com/linecorp/armeria/internal/logging/DropwizardMetricConsumerTest.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+            throw new UnsupportedOperationException();
+        }
+    }
+
+    private class DummyRpcRequest implements RpcRequest {
+
+        private final String method;
+
+        DummyRpcRequest(String method) {
+
```
Comment (trustin) [inline src/test/java/com/linecorp/armeria/internal/logging/DropwizardMetricConsumerTest.java]: Unnecessary empty line

**Comment by trustin** [comment]
One super trivial nit. LGTM besides that. Waiting for @blmarket's comments who was the original author of the metrics stuff.

**Pull request merged**